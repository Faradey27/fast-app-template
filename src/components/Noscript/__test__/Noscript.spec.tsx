import NoscriptDriver from './Noscript.driver';

describe('Noscript component', () => {
  let driver: NoscriptDriver;

  beforeEach(() => {
    driver = new NoscriptDriver();
  });

  it('should render correctly', () => {
    driver.when.created();
    expect(driver.is.ok()).toBeTruthy();
  });

  it('should match snapshot', () => {
    driver.when.created();
    expect(driver.get.component()).toMatchSnapshot();
  });
});
