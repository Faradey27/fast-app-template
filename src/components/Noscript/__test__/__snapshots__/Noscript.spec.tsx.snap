// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Noscript component should match snapshot 1`] = `
<Noscript>
  <noscript
    dangerouslySetInnerHTML={
      Object {
        "__html": "Some text",
      }
    }
    data-testid="noscript-component"
  />
</Noscript>
`;
